### ISSUE TITLE
Usability of the questionnaire

### ISSUE THREAD

**Issue by carolina-ott** [vsaase/covid19selfreport#9]
The idea in gerneral is, to make the questionnaire as easy as possible for everyone. 
I think for the symptoms it might be interesting to get information on how long symptoms were felt and how strong they were. 
This could look like this:

![grafik](https://user-images.githubusercontent.com/62143783/77236407-7939a280-6bbe-11ea-8939-f5f982f32445.png)

**Comment by Skhurana136** [comment]
Did anyone implement this yet? Maybe I could look into it.

**Comment by mctraub** [comment]
> Did anyone implement this yet? Maybe I could look into it.

I don't think so. I looked into it last night, but couldn't figure out how to do that. So, please go ahead. :-)

**Comment by vsaase** [comment]
In psychology questionnaires like that are called "Likert-scale", they usually limit to 5 levels for ease of use

**Comment by vsaase** [comment]
please also test the form on your mobile phone

**Comment by mctraub** [comment]
> In psychology questionnaires like that are called "Likert-scale", they usually limit to 5 levels for ease of use

Yes, but when I was asked to rate my (headache) pain it was always a scale from 0 - 10.
https://en.wikipedia.org/wiki/Pain_scale#Numeric_rating_scale

**Comment by Skhurana136** [comment]
So yes, it's complicated :)
Does anyone have experience with using Dash? I see that one can add a slider using Dash and I don't expect it to be in conflict with other libraries here. It would be something simple such as:
#----
import dash_core_components as dcc

dcc.Slider(
    min=0,
    max=10, #slider from 0 to 10
    step=None,
    marks={
        0: 'keine Beschwerden',
        5: 'mittelmäßig Beschwerden',
        10: 'extrem Beschwerden'
    }, #3 labels for different stages
    value=0, #default start with 0 point on the slider
included=False #do not select the entire slider, but just move the button on it
)  
#-----
Source: Dash: https://dash.plot.ly/dash-core-components/slider 
#----
